packages/slate-react/src/components/editable.tsx

```diff
@@ -98,6 +114,7 @@ export type EditableProps = {
   style?: React.CSSProperties
   renderElement?: (props: RenderElementProps) => JSX.Element
   renderLeaf?: (props: RenderLeafProps) => JSX.Element
+  renderPlaceholder?: (props: typeof defaultPlaceholderProps) => JSX.Element
```

Comment:
Can you refactor this to be declared the same was as `RenderElementProps/RenderLeafProps` for consistency? So people can use those types in their own code if they need to.